osquery/dispatcher/tests/scheduler_tests.cpp

```diff
@@ -173,6 +173,47 @@ TEST_F(SchedulerTests, test_scheduler) {
   TablePlugin::kCacheInterval = backup_interval;
 }
 
+TEST_F(SchedulerTests, test_scheduler_sum_overtime) {
+  auto backup_step = TablePlugin::kCacheStep;
+  auto backup_interval = TablePlugin::kCacheInterval;
+
+  // Start the scheduler now.
+  auto now = osquery::getUnixTime();
+  TablePlugin::kCacheStep = now;
+
+  // Update the config with a pack/schedule that contains several queries.
+  std::string config = R"config(
+  {
+    "packs": {
+      "scheduler": {
+        "queries": {
```

Comment:
Don't use actual queries here. Use something like `select 1 as number`, otherwise your test might fail for unrelated reasons. This is still not a perfect unit test, but the best we can get.

Reply:
It makes sense. Thanks